libp2p/errors.nim

```diff
@@ -55,10 +55,11 @@ proc allFuturesThrowing*[T](args: varargs[Future[T]]): Future[void] =
 
   return call()
 
-template tryAndWarn*(msg: static[string]; body: untyped): untyped =
+template tryAndWarn*(message: static[string]; body: untyped): untyped =
   try:
     body
-  except CancelledError as ex:
-    raise ex
-  except CatchableError as ex:
-    warn "Ignored an error", name=ex.name, msg=msg
+  except CancelledError as exc:
+    raise exc # TODO: why catch and re-raise?
```

Comment:
as I explained in the other PR, this is how you deal with cancellation, it as to be re-raised to propagate

Reply:
OK, thats all good then. We might just do:

```nim
try:
  ...
except CatchableError:
  raise
```

No temp/intermediary var, tho it might get optimized away either way.

Reply:
just don't do anything if you have to do that , just don't handle

Reply:
THat helper is pointless if we remove cancellation, I made it just to make sure cancellation was handled properly.

Reply:
> just don't do anything if you have to do that , just don't handle

I think it's standard behavior, does the same thing as catch-reraise with variable. Not worth the trouble to change anything right now.